calour/plotting.py

```diff
@@ -93,6 +93,9 @@ def plot_enrichment(exp: Experiment, enriched, max_show=10, max_len=40, ax=None,
         name for terms enriched in group1 or group2 respectively, or None to not show legend
     colors: tuple of (str, str) or None (optional)
         Colors for terms enriched in group1 or group2 respectively
+    enriched_exp_color: str or None, optional
+        If not None, the color to show the number of enriched experiments for each term in the bar.
+        None to not show the enriched experiments count
```

Comment:
what does the default 'w' mean?

Reply:
white. the background color is the bar color (red/green) so white looks nice.
added explanation and changed 'w' to 'white' in PR